doc/user/layouts/shortcodes/json-parser.html

```diff
@@ -37,11 +37,20 @@
 /* Helper Methods */
 
 function escapeString(s) {
-    return s.replace(`'`, `''`);
+    return s.replace(/'/g, `''`);
 }
 
 function escapeIdent(s) {
-    return s.replace(`"`, `""`);
+    return `"${s.replace(/"/g, `""`)}"`;
+}
+
+function isDate(value) {
```

Comment:
Fixes the date bug reported here:
https://github.com/MaterializeInc/materialize/issues/25178#issuecomment-1940390196